CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Houston
+**Prepare for liftoff.**
+
+Firstly, thank you for taking the time to contribute to Houston :+1:
```

Comment:
dunno if "firstly" is really needed here, especially since there's no "second" :stuck_out_tongue: